.azp/test.yml

```diff
@@ -14,5 +14,14 @@ schedules:
 variables:
 - template: variables/image.yml
 
+resources:
+  repositories:
+    - repository: asottile
+      type: github
+      endpoint: github
+      name: asottile/azure-pipeline-templates
+      ref: refs/tags/v2.1.0
+
 jobs:
+- template: job--pre-commit.yml@asottile
```

Comment:
So we are running a job that is define by a .yml file in asottile/azure-pipeline-templates. What if this repo changes the file, or goes stale? We just copy the file in to our templates directory and remove the resources block to keep control.

Reply:
It's tagged to a release so this file should not be expected to change. I would prefer to let it use the external template for simplicity. `ref: refs/tags/v2.1.0`